client/tsconfig.json

```diff
@@ -15,6 +15,7 @@
     "noFallthroughCasesInSwitch": true,
     "module": "esnext",
     "moduleResolution": "node",
+    "noImplicitAny": false,
```

Comment:
I don't think this should be set to false. Setting this to true didn't yield any errors, and I don't think we should be reducing the capabilities of the ts compiler to check types in our project, which is likely going to lead to some avoidable errors.